coordinator/migrations/2024-02-12-113100_add_contract_transaction/down.sql

```diff
@@ -0,0 +1,4 @@
+DROP TABLE if exists trade_params;
+DROP TABLE if exists contract_transactions;
+
+DROP TYPE IF EXISTS "Contract_Transactions_State_Type";
```

Comment:
I love this comment: 

🙃 missing newline

Reply:
8f2c78a60fbf3c01975995fee6239a91873344d1